quarkchain/cluster/grpc_client.proto

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
should be using the one defined in another repo as source of truth (once the latest PR has been merged)